scilpy/tracking/utils.py

```diff
@@ -126,6 +126,11 @@ def add_seeding_options(p):
                                     help='Number of seeds per voxel.')
     seed_sub_exclusive.add_argument('--nt', type=int,
                                     help='Total number of seeds to use.')
+    seed_sub_exclusive.add_argument(
+        '--in_custom_seeds', type=str,
+        help='Path to a file containing a list of custom seeding \n'
+             'coordinates (.txt, .mat or .npy). They should be in'
```

Comment:
in  (add a space)

Do you really support .txt, .mat, .npy format? All three? Should you provide an example in the doc? @arnaudbore 
Is this just a file with an x_i y_i z_i on each row? 

Providing a little more info seems important for the user. Otherwise, the user will have to go into the code

Reply:
I'm using a function that was already made and used by scilpy called "import_matrix_in_any_format()" 😄
I checked and it supports .txt, .mat and .npy.

As for the way to format the seeds, as long as numpy interprets it as an (n, 3) array.
mat and npy should be automatic, but if you want to make your own seeds with a .txt file, yes you have to do:

[12.23, 126.316, 15.5]
[13.23, 127.316, 16.5]
[14.23, 128.316, 17.5]
...

I could mention this for the user